pkg/controllers/managementuser/alert/watcher/cluster_scan.go

```diff
@@ -50,7 +50,7 @@ func (csw *ClusterScanWatcher) Sync(_ string, cs *v3.ClusterScan) (runtime.Objec
 	if cs == nil {
 		return nil, nil
 	}
-	if cs.DeletionTimestamp != nil {
+	if cs.DeletionTimestamp != nil || cs.Status.CisScanStatus == nil {
```

Comment:
Let's add a comment on why we skip the object if `CisScanStatus` is nil

Reply:
Will do.

Reply:
Done.